### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by jasper-** [0xcafed00d/tracetcp-go#1]
Hello,
Thanks for this great tool. 
Is it an option to build in IPv6 support?

Thanks.

**Comment by 0xcafed00d** [comment]
Yes, IPv6 support is something I should look at adding at this point, stay tuned!
